### PR TITLE
feat: Update analytics instructions with the new flow gf-406

### PR DESCRIPTION
**Pull request by s1rserg**
* Updated analytics instructions.
* Updated margins to not have scroll in modal on full hd displays.
* Created scripts frontend module.
image

### CODE REVIEW

**Review by what1s1ove — APPROVED** [review]

File: apps/frontend/src/pages/project/libs/components/setup-analytics-modal/setup-analytics-modal.tsx [hunk 1]
```diff
@@ -143,12 +158,46 @@ const SetupAnalyticsModal = ({
 							<span className={styles["list-item-title"]}>Node.js 20</span>: The
 							script requires Node.js 20 to be installed on your machine.
 						</li>
+						<li>
```
Comment (liza-veis) [inline apps/frontend/src/pages/project/libs/components/setup-analytics-modal/setup-analytics-modal.tsx:161]: I think we should also mention that for now only unix based systems are supported

**Review by liza-veis — CHANGES_REQUESTED** [review]

**Comment by liza-veis** [comment]
also instead of `Copy the command below and modify with your local repository's path:`, let's say explicitly: `Copy the command below and replace <project-page> placeholder with your local repository's path:`

File: apps/frontend/src/pages/project/libs/components/setup-analytics-modal/setup-analytics-modal.tsx (hunk 1 above)
Comment (s1rserg) [reply apps/frontend/src/pages/project/libs/components/setup-analytics-modal/setup-analytics-modal.tsx:161]: should it be in prerequisites or in installation steps title in brackets 
image
image

Comment (liza-veis) [reply apps/frontend/src/pages/project/libs/components/setup-analytics-modal/setup-analytics-modal.tsx:161]: Looks good to me now

**Review by liza-veis — APPROVED** [review]

**Pull request merged**